dtfs-central-api/src/helpers/calculate-fixed-fee-from-days-remaining.ts

```diff
@@ -0,0 +1,34 @@
+import Big from 'big.js';
+import { CalculateFixedFeeFromDaysRemainingParams } from '@ukef/dtfs2-common';
+
+/**
+ * An admin fee (fixed at 10%) is applied to the fixed fee, meaning
+ * we subtract this amount when calculating it
+ */
+const BANK_ADMIN_FEE_ADJUSTMENT = 0.9;
+
+/**
+ * calculateFixedFeeFromDaysRemaining
+ * Calculates the fixed fee for the given parameters and number of days provided
+ * @param param - The parameters to calculate the fixed fee with
+ * @param {Number} param.utilisation - The facility utilisation
+ * @param {Number} param.numberOfDaysRemainingInCoverPeriod - The number of days remaining in the cover period
+ * @param {Number} param.interestPercentage - The facility interest percentage
+ * @param {Number} param.dayCountBasis - The facility day count basis
+ * @returns {Number} The fixed fee for the current report period
```

Comment:
`@param {number}`?

Can't see any other instances of this in uppercase

Reply:
changed to number